be/src/types/bitmap_value.cpp

```diff
@@ -898,28 +889,35 @@ void BitmapValue::compress() const {
 }
 
 void BitmapValue::clear() {
-    _type = EMPTY;
     if (_bitmap != nullptr) {
         _bitmap->clear();
     }
+    if (_set != nullptr) {
+        _set->clear();
+    }
+    _sv = 0;
+    _type = EMPTY;
+}
+
+void BitmapValue::reset() {
+    _bitmap.reset();
     _set.reset();
     _sv = 0;
+    _type = EMPTY;
 }
 
-void BitmapValue::_convert_to_smaller_type() {
-    if (_type == BITMAP) {
```

Comment:
it is safe to remove this if ?

Reply:
Yes，it's safe, Currenty ,only bitmap type will use the function